client/app/house/components/HouseUpdateForm.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+  Input,
+  Button,
+  RadioGroup,
+  RadioGroupItem,
+} from '@/_components/ui'
+import { HouseType } from '@/_types'
+import { DynamicRuleField } from './DynamicRuleField'
+import { useUpdateHouse } from '../hooks'
+
+type HouseUpdateFormProps = {
+  defaultData: HouseType
+}
+
+const HouseUpdateForm = ({ defaultData }: HouseUpdateFormProps) => {
```

Comment:
I could make one component for create and update, but because of the rules, it's too much condition I had to make so I decided to make in separate component (It's more simple for this house setting)